onnxruntime/python/tools/kernel_explorer/kernels/gemm_ck.cc

```diff
@@ -35,77 +36,45 @@ struct DataTypeAdaptor<half> {
   using type = ck::half_t;
 };
 
+}  // namespace
```

Comment:
Move this header (#include "ck/library/tensor_operation_instance/gpu/gemm.hpp") before ck/tensor... ?